test/services/builder_base_service_test.rb

```diff
@@ -0,0 +1,78 @@
+# typed: false
+# frozen_string_literal: true
+
+require "test_helper"
+
+class BuilderBaseServiceTest < ActiveSupport::TestCase
+  test "BuilderBaseService.call" do
+    assert_raises NotImplementedError do
+      BuilderBaseService.call
+    end
+  end
+
+  test "UseBuildBaseService" do
+    class UseBuildBaseService < BuilderBaseService
```

Comment:
@tsmartt I added a basic example of using `build` in the single method context as we discussed.